### DISCUSSION TITLE
A question about if the limitations like maximum amount of memory (12 GB) were solved

### DISCUSSION THREAD

**Discussion by zhou9240** [CERN/TIGRE#585] [Q&A]
Dear all,

Would you please tell me if the limitations described by the first release of TIGRE were solved? 
The description about the limitations from the paper is shown below:
"Currently, TIGRE does not support multi-GPU architectures. A further weakness of the toolbox is the small number of functions for data loading and post-processing.....Another limitation of TIGRE comes from the current limitations in GPU technology. Currently, 12 GB is the maximum amount of memory on a GPU board..."

Thanks
Zhongxing

**Comment by AnderBiguri** [comment]
Indeed, this has bee solved since 5 years ago. You can now use any gpu and
multi gpu is supported.
Ander

**Reply by zhou9240** [reply]
Thank you!
